docs/user/spark.rst

```diff
@@ -0,0 +1,86 @@
+GeoMesa Spark
```

Comment:
Yeah, this is great! Make sure you add an entry to ``docs/user/index.rst`` so that this new chapter is included in the manual build.